servicemigration/metrics.py

```diff
@@ -15,6 +15,8 @@ def deserialize(data):
     """
     Parse data into Metrics.
     """
+    if not data:
```

Comment:
I think this is too broad. I suggest returning an empty list instead. `None` is not an expected return value from this function, for example.